### DISCUSSION TITLE
The ability to embed the OQRS form into a website

### DISCUSSION THREAD

**Discussion by DM5CB** [wavelog/wavelog#583] [Ideas]
The possibility to embed the OQRS form in a website without embedding the entire page as an IFrame, but only the box with the query form

**Comment by HB9HIL** [comment]
We had this topic in the past :) 👍 

I think we got that on our list

**Reply by DM5CB** [reply]
Sorry for my question but i could find any topic like that with the search function here and in Issues.

**Reply by HB9HIL** [reply]
No Problem!

**Comment by HB9HIL** [comment]
@DM5CB 

See PR #616
